src/progress.jl

```diff
@@ -16,9 +13,10 @@ end
 Create a new progress bar and register it with Juno, if possible.
 """
 function ProgressBar(;name = "", msg = "")
-  id = string(Base.Random.uuid1())
-  p = ProgressBar(name, "", id, 0.0, msg)
+  p = ProgressBar{Val{isactive()}}(string(Base.Random.uuid1()))
```

Comment:
Heh, just wanted to abuse multiple dispatch for the fallback ;) 
But yeah, I'll revert to `if isactive() ... else ...` on every call.